examples/connectivity/plot_adhd_covariance.py

```diff
@@ -8,11 +8,26 @@
  
 plotted_subject = 0  # subject to plot
 
+import numpy as np
 
 import matplotlib.pyplot as plt
+
+from nilearn import plotting, image
 from nilearn.plotting import cm
 
 
+def plot_connectome(cov, atlas_maps, **kwargs):
```

Comment:
>> Also there is already a function to plot the matrices in this example.

> I don't like that either :).

Given that we advise people to install anaconda, they shoulds have statsmodel installed. We could use statsmodel function for that.